features/networking/ovn-egress-ip.feature

```diff
@@ -419,6 +427,8 @@ Feature: OVN Egress IP related features
     #Check events
     Then I wait up to 30 seconds for the steps to pass:
     """
+    Given I switch to cluster admin pseudo user
+    And I use the "default" project
```

Comment:
The added steps can be put outside of 'wait up to', as no need to duplicate those steps.

Reply:
Thanks @liangxia ,  you are right, updated it!